ui/components/app/desktop-enable-button/desktop-enable-button.component.js

```diff
@@ -78,9 +89,19 @@ export default function DesktopEnableButton() {
       return;
     }
 
+    trackEvent({
+      category: EVENT.CATEGORIES.DESKTOP,
+      event: 'Desktop Button Clicked',
```

Comment:
do you need to fire an event on the button when you are already firing an event on the state change `DESKTOP_UPDATE_SETTINGS_EVENT`?  seems redundant, so i'd suggest removing the click event here (which is typically used when there is no other event being triggered), but i dont have a _**strong**_ opinion here. :)